kerckhoff/packages/operations/google_drive.py

```diff
@@ -86,6 +86,14 @@ def list_folder(
                     break
 
             except RequestException:
+                error_msg = response.json()["error"]
+                error_code= error_msg["code"]
+
+                if error_code == 404:
+                    logger.error(
+                        f"File not found for id:{gdrive_folder_id} {response.json()}"
+                    )
+                    raise GoogleDriveFileNotFound(response.json()) 
```

Comment:
I believe there is some whitespace at the end of the line

Tiny niggle, I know, but it's best to keep the codebase free of these